.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: generic
+services:
+- docker
+before_install:
+- docker build --target test --tag my-test-image .
+script:
+- echo "hello world"
+- docker run -e FLASK_APP=todo_app/app -e FLASK_ENV=development -e FLASK_SKIP_DOTENV=True
+  -e token="4c82ed8e16b482daf8c417cc0f1187bf943dc7d3dcbd212d08b39b51a5c58847" -e api_key="38562d78c91237d032fc48f4301cab52"
+  -e TRELLO_BOARD_ID="34" my-test-image tests/int_unit
+- docker run -e FLASK_APP=todo_app/app -e FLASK_ENV=development -e FLASK_SKIP_DOTENV=True -e token=$token -e api_key=$api_key -e done_status=$done_status -e doing_status=$doing_status -e toDo_status=$toDo_status -e TRELLO_BOARD_ID=$TRELLO_BOARD_ID my-test-image tests/e2e
```

Comment:
You shouldn't be passing real status or board ids to the e2e test because it should be creating its own temporary board.